tests/test_monitors.py

```diff
@@ -154,7 +155,20 @@ def test_unwanted_httpcodes_should_fail(make_data):
 def test_unwanted_httpcodes_should_pass(make_data):
     """Unwanted HTTP Code should pass if # off responses with error status
     codes is lower than expected """
+
+    # Scenario # 1
+    data = make_data({SPIDERMON_UNWANTED_HTTP_CODES_MAX_COUNT: 16})
```

Comment:
What do you think of covering the same scenarios in `test_unwanted_httpcodes_should_fail`?